terms.adoc

```diff
@@ -0,0 +1,130 @@
+////
+Apply pseudo legal capitalization to the main document:
+cat eclipse_development_process.adoc | sed -E -e "`grep -P -oh "^.+(?=\s+\:\:)" terms.adoc | while read -r term; do echo -ne "s|$term|$term|gI;"; done`"
+////
+
+[glossary]
+
+Architecture Council ::
+The Eclipse Architecture Council (AC) is a council of experienced Eclipse committers responsible for identifying and tackling any issues that hinder continued technological success and innovation, widespread adoption, and future growth of Eclipse technology.
+
+Board of Directors ::
+The business and technical affairs of the Eclipse Foundation are managed by or under the direction of the Eclipse Board of Directors. 
+
+Check Point Review ::
+Check Point Review is a category of Review that includes: Creation Review, Progress Review, Release Review, and Graduation Review.
+
+Committer ::
+A committer is a software developer who has the necessary rights to make decisions regarding a Project.
+
+Contributor ::
+A Contributor is anybody who makes contributions to the project.
+
+Creation Review ::
+A Creation Review is a type of Review that marks a Project's transition from the Proposal Phase to the Incubation Phase.
+
+Developer ::
+A Committer or Contributor.
+
+Ecosystem ::
+A commercial Ecosystem is a system in which companies, organizations, and individuals all work together for mutual benefit.
+
+Eclipse Management Organization ::
+The Eclipse Management Organization (EMO) consists of the Eclipse Foundation staff, and the Eclipse Architecture and Planning Councils. 
+
+Graduation Review ::
+A Graduation Review is a type of Review that marks a Project's transition from the Incubation Phase to the Mature Phase.
+
+Incubation Branding ::
+Incubation Branding is the branding that must be applied to all projects in the Incubation Phase.
+
+Incubation Phase ::
+The Incubation Phase is a Phase during which the Project Team develops the process, the community, and the technology for a new Project.
+
+Integration Build ::
+An Integration Build is a build of the project content that is generated periodically (generally weekly) and made available to the Developer community for testing purposes.
+
+Major Release ::
+A Major Release is a type of Release that includes either significant new functionality and/or breaking changes.
+
+Mature Phase ::
+The Mature Phase is a Phase during which a Project operates with an open and transparent process, has an actively involved and growing community, and produces Eclipse-quality technology.
+
+Membership at Large ::
+The Membership at Large (or _Membership_) refers to all members of all types, as defined by the {bylawsUrl}[Bylaws of the Eclipse Foundation].
+
+Milestone ::
+A Milestone is a build of the project content intended for limited distribution to demonstrate progress and solicit feedback. 
+
+Minor Release ::
+A Minor Release is a type of Release that includes new features over a Major Release.
+
+Nightly Build ::
```

Comment:
Where in the EDP is this used? I think we should get rid of this term as many projects don't have nightly builds. By the way, in a worldwide community, there is no such thing as Nightly which has the same meaning for eveyone.

Reply:
It's used in the section on [releases](https://www.eclipse.org/projects/dev_process/#6_4_Releases). I'm all for rewriting that section and coming up with some better words. But I'll need some help if we're going to do this in this iteration.